mapadroid/worker/WorkerBase.py

```diff
@@ -637,13 +637,14 @@ def _ensure_pogo_topmost(self):
                 self._reboot()
                 break
 
-            if self._loginerrorcounter == 2:
+            if self._loginerrorcounter > 1:
```

Comment:
Doesnt this contradict line 612. Reboot after 2+ failures, but restart until 2 failures?!

Reply:
I don't see it. First two failures restart pogo, for the third+ we go forward the extended routine.
Could you explain further what you mean?

Reply:
I agree with @crhbetz.
The alternative would be to not reset the errorcount and then use modulo 2 rather than checking the direct value for equality

Reply:
@crhbetz What I was getting at: When *_loginerrorcount* reaches 2, the code in lines 612-613 triggers a pogo restart. This is immediately followed by a reboot in the block starting on line 640.
This was already the case before your PR, but IMHO this is still an unnecessary step.

Reply:
Got it now, I guess you're right and I changed one more line accordingly :-)